sql/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -689,6 +707,19 @@ public void testCountDistinctFromJoin() throws Exception {
         }
     }
 
+    @Test
+    public void testDescribeOnBatchPortal() throws Exception {
```

Comment:
Can we add a comment or clarify the test name further? It's not obvious how this test works or what it does

Reply:
This is not needed, sorry, will remove